webapp/channels/src/components/file_attachment_list/index.ts

```diff
@@ -24,14 +25,24 @@ export type OwnProps = {
     compactDisplay?: boolean;
     isInPermalink?: boolean;
     handleFileDropdownOpened?: (open: boolean) => void;
+    isEditHistory?: boolean;
+    disableDownload?: boolean;
+    disableActions?: boolean;
 }
 
 function makeMapStateToProps() {
     const selectFilesForPost = makeGetFilesForPost();
 
     return function mapStateToProps(state: GlobalState, ownProps: OwnProps) {
         const postId = ownProps.post ? ownProps.post.id : '';
-        const fileInfos = selectFilesForPost(state, postId);
+
+        var fileInfos: FileInfo[];
+
+        if (ownProps.isEditHistory) {
+            fileInfos = getFilesForEditHistory(state, ownProps.post);
```

Comment:
Not something to change because it's likely not easy, but I wonder about if we should make whoever's using this component provide the `fileInfos` instead of having to have two code paths for getting them from Redux. Given we probably still need a prop to change the functionality like above, I don't think there's much to be gained by doing that though

Reply:
Right, nothing to gain from the change here. Skipping.